service/src/main/resources/application.yml

```diff
@@ -32,9 +32,25 @@ spring:
     password: ${springMailPassword:"none"}
     port: 465
     protocol: 'smtps'
-    dev-mode: true
+    devMode: true
     debug: true
-    test-connection: false
+    testConnection: false
+  template:
+    path: "templates/mail"
+    format: ".html"
+    encoding: "UTF-8"
+    mode: "HTML"
+    cache: false
+  templateNames:
```

Comment:
Would be better if template names would be stored as an enum in code, not in `application.yml`.